hax-lib-protocol/src/cal.rs

```diff
@@ -0,0 +1,194 @@
+//! This module defines a cryptographic abstraction layer for use in
```

Comment:
Given that we want an abstraction, we should have traits to make it easier to implement this again for other backends.
But that's not urgent. Let's open an issue for this.

Reply:
I created #471 to track this.